commit 1d599d14709184b8241808aad0d0e6f85652a252
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Jan 23 17:36:50 2017 +0000

    BCM270X_DT: Add reference to audio_pins to CM dtb
    
    The CM1 dtb contains an empty audio_pins node, but no reference to it.
    Adding the usual pinctrl reference from the audio node enables the
    audremap overlay (and others) to easily turn on audio.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2708-rpi-cm.dts b/arch/arm/boot/dts/bcm2708-rpi-cm.dts
index eb8662f0d222..10be69972bd1 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-cm.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-cm.dts
@@ -85,6 +85,11 @@
 	pinctrl-0 = <&i2s_pins>;
 };
 
+&audio {
+	pinctrl-names = "default";
+	pinctrl-0 = <&audio_pins>;
+};
+
 &hdmi {
 	hpd-gpios = <&gpio 46 GPIO_ACTIVE_HIGH>;
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1133_1d599d14709184b8241808aad0d0e6f85652a252.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1133_1d599d14709184b8241808aad0d0e6f85652a252.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
